app/helpers/dogs_helper.rb

```diff
@@ -0,0 +1,5 @@
+module DogsHelper
+  def index_page_title
```

Comment:
ah, for some reason I thought this would magically need dog before it, should we go `dog_index_page_title` just to somewhat namespace it?

Reply:
no problem will do